packs/actions/devour-ambient-magic.json

```
@@ -0,0 +1,33 @@
+{
+    "_id": "3sgY9afTTBhZXE8V",
+    "img": "systems/pf2e/icons/actions/Reaction.webp",
+    "name": "Devour Ambient Magic",
+    "system": {
+        "actionType": {
+            "value": "reaction"
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "defensive",
+        "description": {
+            "value": "<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> A creature Casts a Spell with you as the only target</p><hr /><p><strong>Effect</strong> Your ostilli radiates cyan light as its tentacle-like filters attempt to consume the magical effect. You can attempt to counteract the triggering spell with an Arcana or Nature check and a counteract rank equal to half your level.</p>"
```

```suggestion
            "value": "<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> A creature Casts a Spell with you as the only target</p><hr /><p><strong>Effect</strong> Your ostilli radiates cyan light as its tentacle-like filters attempt to consume the magical effect. You can attempt to counteract the triggering spell with an @Check[type:arcana] or @Check[type:nature] check and a counteract rank equal to half your level.</p>"
```
Adding some convenient buttons